v0/destinations/singular/util.js

```diff
@@ -19,7 +19,8 @@ const {
   extractCustomFields,
   getValueFromMessage,
   isDefinedAndNotNull,
-  toUnixTimestamp
+  toUnixTimestamp,
```

Comment:
We are not using `toUnixTimestamp` so we can remove this.

Reply:
addressed